src/Controller/ContentManagement/ContentTypeController.php

```diff
@@ -809,8 +819,10 @@ public function editAction($id, Request $request, LoggerInterface $logger)
         $client = $this->getElasticsearch();
 
         try {
+            $env = $contentType->getEnvironment();
+            $index = $env ?  $env->getAlias() : '';
```

Comment:
If env is false you should throw a RuntimeException('Unexpected not found environment')